Dev-Corner/slides/tool_integration.html

```diff
@@ -68,10 +69,762 @@
+                        </script>
+                    </section>
+
+
+                    <section data-markdown>
+                        <script type="text/template">
+                        ### Galaxy tool / wrapper
+
+                        <img src="../images/graphlan_screenshot.png" alt="Tool Form" height=400/>
+
+                        ```bash
+                        graphlan.py --format "png" --size 7 input_tree.txt png_image.png
+                        ```
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                        ### Wrapper
+
+                        - Description of the user interface
+                        - Link between the tool and Galaxy instance
+                            - How to invoke the tool
+                            - What options to pass
+                            - What files the tool will produce as output
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                        ### Wrapper
+
+                        <img src="../images/wrapper_layers.png" height=400/>
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                        ### Wrapper
+
+                        <img src="../images/wrapper_big_picture_1.png" height=400/>
+
+                        <small>[Galaxy Tool XML File](https://docs.galaxyproject.org/en/latest/dev/schema.html)</small>
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                        ### Wrapper
+
+                        <img src="../images/wrapper_big_picture_2.png" height=400/>
+
+                        <small>[Galaxy Tool XML File](https://docs.galaxyproject.org/en/latest/dev/schema.html)</small>
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                        ### Wrapper
+
+                        <img src="../images/wrapper_big_picture_3.png" height=400/>
+
+                        <small>[Galaxy Tool XML File](https://docs.galaxyproject.org/en/latest/dev/schema.html)</small>
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                        ### `command`
+                        #### How to invoke the tool?
+
+                        ```xml
+                        <command><![CDATA[
+                            graphlan.py
```

Comment:
I was just asking if you think its worth to put this into the slides and not use the planemo examples directly.